src/validation.cpp

```diff
@@ -2901,6 +2896,8 @@ bool CChainState::DisconnectTip(BlockValidationState& state, DisconnectedBlockTr
     AssertLockHeld(cs_main);
     if (m_mempool) AssertLockHeld(m_mempool->cs);
 
+    int64_t nTime1 = GetTimeMicros();
```

Comment:
Consider using here `SteadyClock::now()`? It returns time in time units, not int64_t.

`GetTimeMicros` is removed by bitcoin/bitcoin#27233

Reply:
It's better to use `GetTimeMicros` here for consistency sake and replace it with `SteadyClock::now()` in the same report it's replaced for other Dash-specific timers and timers inherited from bitcoin imo.